### ISSUE TITLE
Promotional video or screencast

### ISSUE THREAD

**Issue by PanderMusubi** [Xaviju/inkscape-open-symbols#26]
When the current supported symbol collections function all without errors, a video or screencast could be made. The aim is to draw more attention to the awesome powers of this project and Inkscape and the pakcaging of this project for Debian, Ubuntu etc.

Many graphical designers and other potential users of Inkscape could be motivated to start using Inkscape because of this powerful feature.

**Comment by Xaviju** [comment]
Would be nice, but I will no have time, neither the knowledge to do it correctly. If anyone is interested in helping with this it would be awesome.

**Comment by victorwestmann** [comment]
I agree with this suggestion. +1 vote from me. Which software should one use to make this happen? And what should be recorded? How to use Symbols inside Inkscape?

**Comment by Xaviju** [comment]
Don't know about screencast software (Ubuntu recommends: https://help.ubuntu.com/community/Screencast) but it would be nice to show how to add and use the icons.
English is not my mother tongue so it would be better if an English-speaker helps with this.

**Comment by PanderMusubi** [comment]
A short article for Libre Grahics Magazine and related websited would also be helpful for the project.

**Comment by Xaviju** [comment]
I thought LibreGraphics Magazine was dead after this announcement -> http://lists.libregraphicsmag.com/pipermail/libregraphicsmag-libregraphicsmag.com/2016-March/000213.html isn't it?

If not, can you point me to a how to contribute?

**Comment by PanderMusubi** [comment]
Ah, too bad. Well, there is also http://libregraphicsworld.org/ and their G+ stream. We could ask them to do an article or provide one.

**Comment by Xaviju** [comment]
@PanderMusubi I'm writting to libregraphicsWorld today.
I'm thinking that it might be interesting to have a simple website as well. :)
